editor/src/components/canvas/controls/select-mode/grid-gap-control-component.tsx

```diff
@@ -220,8 +220,8 @@ const GridRowOrColumnHighlight = (props: {
   beingDragged: boolean
   onMouseOver: () => void
   elementHovered: boolean
-  gridJustifyContent: FlexJustifyContent | null
-  gridAlignContent: AlignContent | null
+  gridJustifyContent: string | undefined
```

Comment:
do we want to keep them untyped? (since for example in the metadata we do use typings for justify/align). we can maybe (later) have typed functions (getJustifyValue, getAlignValue) that return the value from the computed style as the appropriate type

Reply:
These are just matching the values from the computed style, so that we can use them directly.